src/Microsoft.Azure.WebJobs.Host/Bindings/IOrderedValueBinder.cs

```diff
@@ -3,8 +3,14 @@
 
 namespace Microsoft.Azure.WebJobs.Host.Bindings
 {
-    internal interface IOrderedValueBinder : IValueBinder
+    /// <summary>
+    /// Defines an <see cref="IValueBinder"/> that provides an ordering hint./>
+    /// </summary>
+    public interface IOrderedValueBinder : IValueBinder
```

Comment:
We didn't want to make this public, but I don't see any other way. The ServiceBus bindings use it.